code/index.html

```diff
@@ -1,21 +1,118 @@
 <!DOCTYPE html>
 <html>
-
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <title>Page about myself</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
     <!-- If you're using google fonts, that should be added here -->
   </head>
+
+  <style>
+    body {
+      margin: 0;
+    }
+
+    div {
+      padding: 65px;
+    }
+
+    h1 {
+      text-align: center;
+      color: rgb(5, 39, 5);
+    }
+    h2 {
+      text-align: left;
+      color: rgb(5, 39, 5);
+    }
+
+    .top {
+      background-image: url(images/algues.jpeg);
+      /*background-color: rgb(144, 232, 144);*/
+      font-style: white;
+      /* color: rgb(241, 245, 249);*/
+    }
+
+    .middle {
+      background-color: white;
+    }
+
+    .down {
+      background-color: #111f11;
+    }
+  </style>
```

Comment:
Better to keep all the CSS to the CSS file so you have it all in one place, away from the HTML 🙌